js/Track/Model/File/BED.js

```diff
@@ -4,13 +4,29 @@ Genoverse.Track.Model.File.BED = Genoverse.Track.Model.File.extend({
 
     for (var i = 0; i < lines.length; i++) {
       var fields = lines[i].split('\t');
-
-      if (fields.length < 3) {
+     
+      if (fields.length < 3 || fields[0] == 'track' || fields[0] == 'browser') {
         continue;
       }
+      console.log(fields);
+      var len = fields.length;
 
       if (fields[0] == chr || fields[0].toLowerCase() == 'chr' + chr || fields[0].match('[^1-9]' + chr + '$')) {
-        var score = parseFloat(fields[4], 10);
+        var feature = {};
+        feature.chr = chr;
+        feature.start = parseInt(fields[1], 10);
+        feature.end   = parseInt(fields[2], 10);
+        feature.name  = fields[3];
+
+        if (len > 3) feature.score = parseFloat(fields[4], 10);
+        if (len > 5) feature.strand = fields[5];
+
+        if (len > 7) {
+          feature.thickStart = fields[6];
+          feature.thickEnd   = fields[7];
+          if(feature.thickEnd == feature.thickStart == feature.chromStart) feature.drawThickBlock = false;
```

Comment:
feature.chromStart has not been set - did you mean feature.start?
Also, feature.drawThickBlock is never used anywhere.